src/server/utils/prepare.js

```diff
@@ -10,21 +10,43 @@ var exec = require('child_process').exec,
  * @return {Promise}
  */
 function execCordovaPrepare(projectRoot, platform) {
-    var deferred = Q.defer();
+    return retryAsync(getExecCordovaPrepareImpl(projectRoot, platform), 2);
```

Comment:
Thank you @TimBarham ! I forgot to mention that these changes are not stopper for merging this change :)
I think is great to move the `retryAsync` function to `jsUtils`.